src/boat_simulator/boat_simulator/nodes/physics_engine/physics_engine_node.py

```diff
@@ -278,6 +278,7 @@ def __init_timer_callbacks(self):
 
     # PUBLISHER CALLBACKS
     def __publish(self):
+        self.__update_boat_state()
         """Synchronously publishes data to all publishers at once."""
```

Comment:
Could you please move the method call so that it appears just below the docstring

Reply:
I have changed this in the new commit.